frontend/public/components/routes.tsx

```diff
@@ -295,6 +249,11 @@ const RouteDetails: React.SFC<RoutesDetailsProps> = ({obj: route}) => <React.Fra
       </div>
     </div>
   </div>
+  <div className="co-m-pane__body">
+    <Conditions obj={(route.status && route.status.ingress)
+      ? _.flatten(route.status.ingress.map((ingress)=>ingress.conditions))
```

Comment:
We should have separate tables for each ingress instead of flattening. You need to know which hostname each of the conditions are for, so the hostname should be in the header above each table.

Reply:
So would it be something like 
"www.daves-great-app.com" or
"www.daves-great-app.com Conditions" or 
"Conditions for www.daves-great-app.com"
as the table title?  Or have the hostname as a sub-heading, something like:
"Conditions
www.daves-great-app.com  <-- smaller text
---table---"

Reply:
Looking at the YAML, the other items are important, too.

```yaml
    - host: console-tectonic-console.svc.ci.openshift.org
      routerName: router
      conditions:
        - type: Admitted
          status: 'True'
          lastTransitionTime: '2018-05-23T16:28:52Z'
      wildcardPolicy: None
```

I would suggest something like


console-tectonic-console.svc.ci.openshift.org (header)

Router: router
Wildcard Policy: None

(conditions table)

Reply:
Hi, I don't think I've seen any 'label: value' side-by-sides in the UI, it's always label ontop of value.  This is what I have so far:
![image](https://user-images.githubusercontent.com/12733153/41557350-5ee9e14a-730b-11e8-93d5-55f5f5fe18f6.png)
- It seems like we're missing a table title still, something like 'Conditions'

Reply:
I do see this in nodes, should I do the same stying for Router & Wildcard Policy?
![image](https://user-images.githubusercontent.com/12733153/41558204-13991c58-730e-11e8-91c1-20d6ad527871.png)
Meaning grey for the label, and black for the value, side-by-side?  -thanks

Reply:
cc @beanh66  :-)

Reply:
I would use the existing style. I'd probably add "Conditions" above the table as well